pyproject.toml

```diff
@@ -1,3 +1,41 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# Testing tools configuration
+[tool.coverage.run]
+branch = true
+
+[tool.coverage.report]
+fail_under = 85
```

Comment:
We're currently at 99 percent coverage. Can we set it to that to avoid regressions?